cli-plugins/apps/create-workload.hbs.md

```diff
@@ -36,11 +36,11 @@ Where:
 - `--git-branch` (optional) specifies which branch in the repository to pull the code from.
 - `--type` is used to distinguish the workload type.
 
-You can find the options available for specifying the workload in the command reference for [`workload create`](command-reference/tanzu-apps-workload-create.md), or you can run `tanzu apps workload create --help`.
+You can find the options available to add the workload specifications by running `tanzu apps workload apply --help`.
```

Comment:
View the full list of supported workload configuration options by running `tanzu apps workload apply --help`.